src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -339,7 +346,8 @@ export const AssetPriceDetails = ({ asset, collection }: AssetPriceDetailsProps)
         <CollectionNameContainer>
           <DefaultLink to={`/nfts/collection/${asset.address}`}>
             <CollectionHeader>
-              {collection.collectionName} {collection.isVerified && <VerifiedIcon />}
+              {collection.collectionName}
+              <VerifiedIconContainer>{collection.isVerified && <StyledVerifiedIcon />}</VerifiedIconContainer>
```

Comment:
<img width="407" alt="Screen Shot 2022-12-05 at 11 07 11 AM" src="https://user-images.githubusercontent.com/15934595/205689179-9b058741-bb9c-43c4-a586-149646b725a7.png">


The above is what this looks like with no styling.  I tried a bunch of ways to fix it... margin-top, padding-top, flex align-items:center .. etc. this was the only one I could get to work